### PR TITLE
Csharp running multiple files

### LINKED ISSUE #38: Run csharp with multiple files
- [x] Research multiple ways to run multiple csharp files 

- [x] Compile multiple files with csharp

- [x] Run multiple files with csharp

### PR DESCRIPTION
**Pull request by Snip-77**
Implemented compiling and running multiple files in Csharp. Modified the Command building class.

### CODE REVIEW

File: src/main/java/org/jalasoft/moi/model/csharp/CsharpHandler.java [hunk 1]
```diff
@@ -29,7 +29,7 @@
  */
 public class CsharpHandler implements IHandler {
 
-    private static final String FILE_RELATIVE_PATH = "C:\\Users\\Admin\\Documents\\temp\\";
+    private static final String FILE_RELATIVE_PATH = "C:\\Users\\m-a-u\\Documents\\Jala\\Git\\Pruebas cmd csharp";
```
Comment (luqp) [inline src/main/java/org/jalasoft/moi/model/csharp/CsharpHandler.java]: A more consistent name for the variable can be ABSOLUTE_PATH

File: src/main/java/org/jalasoft/moi/model/csharp/CsharpCommandBuilder.java [hunk 2]
```diff
@@ -13,40 +13,27 @@
...
-     * @return String of the command builded with the path Params
+     * @path completePath contains the location of the directory of the files
+     * @return String of the command builded with the path received
      */
     @Override
-    public String buildCommand(Path path) {
-        return getFolderPath(path)+" && "+COMPILER_PATH+path.getFileName().toString()+" && "+getCompiledName(path);
+    public String buildCommand(Path completePath) {
+        folderPath = completePath.toString();
+        return MOVE_TO + folderPath + " && " + COMPILER_PATH + COMPILE_ALL_AND_OUTPUT + " && " + RUN_OUTPUT;
```
Comment (luqp) [inline src/main/java/org/jalasoft/moi/model/csharp/CsharpCommandBuilder.java]: I believe that you should put this `completePath.toString()` directly into the command
`return MOVE_TO + completePath.toString() + " && " + CO...`

**Review by nanrehd — DISMISSED** [review]

File: thirdparty/csharp/Local/NameFile2.cs [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+{
+public class file2 
+{
+    public void go() 
+    {
+        Console.WriteLine("File 2!!!");
+    }
+}
+
+}
```
Comment (luqp) [inline thirdparty/csharp/Local/NameFile2.cs]: No newline at end of file

File: thirdparty/csharp/Local/mainCsharp.cs [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+public class MainClass 
+{
+    public static void Main(string[] args) 
+    {
+        Console.WriteLine("Hello World1");
+        file2 user = new file2();
+        user.go();
+        Console.WriteLine("Hello World2");
+        //Console.ReadKey();
```
Comment (luqp) [inline thirdparty/csharp/Local/mainCsharp.cs]: There should be no code comments when merging some code

**Review by nanrehd — DISMISSED** [review]
Please don't forget punctuation.

**Review by carlosmc23 — DISMISSED** [review]

**Review by nanrehd — APPROVED** [review]

**Pull request merged**